README.md

```diff
@@ -506,6 +514,17 @@ This allows to identify build errors caused by smalltalkCI updates and
 therefore helps to improve the service. It is possible to opt-out by using the
 `--no-tracking` option.
 
+### Pharo-specific Options
+
+Pharo added some specific options to customise its builds: 
+
+```yml
+env:
+  global:
+  - PHARO_VM=stable|latest              # choose stable/latest VM (default: stable)
```

Comment:
that will create a lot of possibilities for each version of Pharo (not to say that the distinction between itimer and threaded are just available to linux).
honestly, for the moment I would prefer to keep it this way... like "extras" you can add...